### ISSUE TITLE
On the latest version I can't open posts by either tapping or long holding

### ISSUE THREAD

**Issue by kalenpw** [Kinark/winston#14]
I assume this is not happening to only me, but I can provide more detailed specs if needed.

On the most recent version I can not open a post's comments by either tapping the post or long holding and selecting open post

**Comment by ojsef39** [comment]
yes more details would be great, my first assumption would be that you are trying to open a post from search. (see https://discord.com/channels/1094807340903039048/1140840231143215104)

if this is not the issue youre seaing please provide more info like a screen recording etc.
